crawling/main.py

```diff
@@ -0,0 +1,68 @@
+import time
+from apscheduler.triggers.date import DateTrigger
+import process_division
+import database.insert_db, database.delete_db
+import schedule
+from apscheduler.schedulers.background import BackgroundScheduler
+from datetime import datetime
+
+def job():
+    data_list = []
+    divisions = [i for i in range(1, 18)]
+
+    # 어제의 데이터 삭제
+    database.delete_db.delete_data()
+
+    for i in divisions:
+        data_list.extend(process_division.process_division(i, initial=1))
+
+    print("final: ", len(data_list))
+    print("--- %s 초 ---" % (time.time() - start_time))
+
+    database.connect_db.insert_data(data_list)
+
+    print("--- %s 초 ---" % (time.time() - start_time))
+
+def job_for7days():
+    data_list = []
+    divisions = [i for i in range(1, 18)]
+
+    for i in divisions:
+        data_list.extend(process_division.process_division(i, initial=7))
+
+    print("final: ", len(data_list))
+    print("--- %s 초 ---" % (time.time() - start_time))
+
+    database.connect_db.insert_data(data_list)
+
+    print("--- %s 초 ---" % (time.time() - start_time))
+
+
+
+if __name__ == '__main__':
+    start_time = time.time()
+    data_list = []
+
+    # 7일치 크롤을 위한 스케줄러 생성(1회만 실행 후 종료)
+    scheduler = BackgroundScheduler()
+
+    # 실행 시간 설정
+    """변경해야 하는 부분. 최초 크롤링에 대한 시간 설정"""
+    run_time = datetime(2024, 8, 7, 10, 53)
+
+    scheduler.add_job(job_for7days, trigger=DateTrigger(run_date=run_time))
+    scheduler.start()
```

Comment:
job_for7days는 최초 1회 실행이라,
scheduler 안쓰고, 바로 함수 호출하면 안 되는건지 궁금합니다!
7일 job과 1일 job이 겹칠까봐 그런건가요?

```suggestion
    job_for7days()
```

Reply:
코드가 서버에 올라간 이후에 어떻게 작동하는지 잘 모르겠어서요. 혹시 main 파일이 실행되고 종료된 후에 다시 실행되어 불필요한 작업을 다시 하게 될까봐 1회만 실행되도록 하였는데요, 그런 경우가 없다면 말씀하신 것처럼 바로 함수 호출해도 되는데 수정해도 괜찮을까요?

Reply:
서버도 파이썬 실행하는건 로컬 환경이랑 똑같이 돌아 갑니당!
python main.py 로 한번 실행 시키면 계속 돌아갑니다!

Reply:
고치신 거 확인했습니다!👍